src/rgw/motr/gc/gc.cc

```diff
@@ -25,6 +26,7 @@ void *MotrGC::GCWorker::entry() {
   uint32_t my_index = \
     ceph::util::generate_random_number(0, motr_gc->max_indices - 1);
   uint32_t rgw_gc_processor_period = cct->_conf->rgw_gc_processor_period;
+  uint32_t rgw_gc_processing_time = cct->_conf->rgw_gc_processor_max_time;
```

Comment:
On line no. 44 we have fetched this again, can you please remove it and use the variable defined here.
```
// Lock has been aquired, start the timer
    std::time_t start_time = std::time(nullptr);
    std::time_t end_time = start_time + \
			 cct->_conf->rgw_gc_processor_max_time - 10;
```

Reply:
done